### DISCUSSION TITLE
324-short

### DISCUSSION THREAD

**Discussion by brusherru** [spacemeshos/sm-net#21] [Networks]
A variant of 324 with 2h epoch.

Besides bug fixes and small improvements, this release is the first one with native apple silicon support. Besides that, we're also adding AppImage support for Linux releases.

We have also switched to Blake3 from sha256 for hashing to unify our hashing algorithms.

Smapp release: https://github.com/spacemeshos/smapp/releases/tag/v0.2.12

### Known issues:
* https://github.com/spacemeshos/go-spacemesh/issues/4067
* smashing initialization during genesis phase causes node to loop

**Comment by pigmej** [comment]
We're aware of: https://github.com/spacemeshos/go-spacemesh/pull/4092

We are preparing the update to ship the fix via the update.

**Comment by pigmej** [comment]
Because of changes in the POST algorithm, the update cannot land this time. Therefore please wait for the updates.
